en/docs/manage/icp/quick-start.md

```
@@ -0,0 +1,148 @@
+import wso2/icp.runtime.bridge as _;
+```
+
+### Start the runtime
+
+```bash
+bal run
+```
+
+On startup, the bridge logs a confirmation:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)